### DISCUSSION TITLE
Interested in learning more about how to contribute to the new maintainers course

### DISCUSSION THREAD

**Discussion by a-ridley** [samucodesh/intro#2] [General]
Hello, I am looking to see how I can contribute which is in relation to the following comment:
https://github.com/open-sauced/intro/issues/116#issuecomment-2099097942 :)

**Comment by samucodesh** [comment]
Hello, @a-ridley  I would like you to read the [CONTRIBUTING file in Spanish](https://github.com/samucodesh/intro/blob/esintrooss/docs/intro-to-oss/translations/es/CONTRIBUTING.md), and you will create the step by step of what we are going to be working on in the new course

**Comment by samucodesh** [comment]
I mean, what will be the order in which we are going to translate [the course](https://github.com/samucodesh/intro/tree/main/docs/becoming-a-maintainer)?

**Comment by samucodesh** [comment]
If that's okay with you, can we connect to a video call to coordinate this part? You can schedule me on my website if it is Monday to Friday. Otherwise we can coordinate by email for Saturdays and Sundays. You can also contact me privately on LinkedIn or on the OpenSauced discord 
![Screenshot 2024-05-07 16 04 34](https://github.com/samucodesh/intro/assets/108969323/014f13bc-b28e-439a-b264-29386d70afb1)

**Reply by a-ridley** [reply]
I counted the files and it looks like there are a total of 16 markdown files in the [becoming-a-maintainer](https://github.com/a-ridley/intro/tree/main/docs/becoming-a-maintainer) course. I was thinking we can split it 8 files each. 

I'll have a branch where I work on these 8 files.

1. README.md
2. _navbar.md
3._sidebar.md
4. additional-resources.md
5. building-community.md
6. communication-and-collaboration.md
7. getting-practical.md
8 .glossary.md



You can have a branch working on these next 8 files.
1. how-to-setup-your-project.md
2. intro.md
3. issues-and-pull-requests.md
4. maintainer-powerups.md
5. maintainers-guestbook.md
6. maintaining-code-quality.md
7. metrics-and-analytics.md
8. your-team.md

Then I will make a PR into your branch when my files have been translated. Once all files are in your branch then you can make a PR to the main repo :) 

Does that work?

**Comment by samucodesh** [comment]
I recommend that we use your repository as "main", it seems like a good option for you to code review me and I code review you

The first thing would be that you will add the `translations/es/` folder

**Comment by samucodesh** [comment]
whenever you need help you can mention me

**Reply by a-ridley** [reply]
It looks like this PR you have https://github.com/open-sauced/intro/pull/115/files has the `translations/es` folder and is ready to be merged. If you can get this merged in soon, then I don't need to re-create this folder. I can just pull from main and start working from there :) 
> The first thing would be that you will add the translations/es/ folder

**Reply by samucodesh** [reply]
You're confused, the course structure changed a few months ago, that's why I haven't updated that branch since then. Now the new structure indicates that both courses must have their own translations folder

**Reply by samucodesh** [reply]
> It looks like this PR you have https://github.com/open-sauced/intro/pull/115/files has the `translations/es` folder and is ready to be merged. If you can get this merged in soon, then I don't need to re-create this folder. I can just pull from main and start working from there :)
> 
> > The first thing would be that you will add the translations/es/ folder

For the new changes we will use the [esintrooss branch](https://github.com/samucodesh/intro/tree/esintrooss) as a reference

**Reply by a-ridley** [reply]
Oh okay, that is not clear when going through that specific PR's comments.  I understand now. I will add a `translation/es` folder and get started! 

I also made sure to read the following guide: https://github.com/open-sauced/intro/blob/main/docs/i18n-guidelines.md#adding-translations

Thanks!

**Reply by samucodesh** [reply]
I have almost 7 months of experience in the project, if you need more context you can ask me

**Reply by samucodesh** [reply]
You can see how I made the changes in the introductory course for the sidebar and navbar

**Reply by samucodesh** [reply]
literally, I made the new changes today in the morning at 2 AM

**Reply by a-ridley** [reply]
okay great, I have started. You can find me on the opensauced discord under this username ltramen
<img width="341" alt="image" src="https://github.com/samucodesh/intro/assets/18725916/927584c0-0ef5-4c0d-9eb4-74b11bfe2380">

**Reply by samucodesh** [reply]
don't pay attention haha, I found a bug and I already fixed it in the [estranslations branch](https://github.com/samucodesh/intro/tree/estranslations)

**Comment by samucodesh** [comment]
currently we can take [estranslations](https://github.com/samucodesh/intro/tree/estranslations)as a reference again!

**Comment by samucodesh** [comment]
@a-ridley  I was looking at your contribution and I realized something very important: the file names must also be translated and must not have accents, use estranslations as a reference
then you will also have to modify the link paths in the README file
